docs/cmdline-opts/write-out.d

```diff
@@ -93,6 +93,9 @@ When an HTTP request was made without --location to follow redirects (or when
 --max-redir is met), this variable will show the actual URL a redirect
 \fIwould\fP have gone to. (Added in 7.18.2)
 .TP
+.B referer
+The referrer header, if there was any. (Added in 7.76.0)
```

Comment:
I'd stick to referer spelling (one r) throughout so it's less confusing

Reply:
My rule of thumb was to use the rr flavour in human sentences and the other one in code. (this also _kind-of_ matches existing use). In this particular case we can work it around by going `The Referer: header, if there was any. (Added in 7.76.0)`. This style can be found in `CURLOPT_AUTOREFERER` and `CURLOPT_REFERER`-related docs. Does that sound OK?

Reply:
Both spellings seem to be used, see [--referer](https://curl.se/docs/manpage.html#-e) using referrer for example. Personally I find it confusing but there seems to be some precedent for it.

Reply:
Yes, it isn't used fully consistently as it is. I've pushed an update for this specific line here, which I'm hoping makes it look better.